src/main/resources/org/jenkinsci/plugins/ewm/steps/ExwsAllocateStep/help-selectedRun.html

```diff
@@ -0,0 +1,18 @@
+<div>
+    <p>
+        If you provide this parameter, the <i>exwsAllocate</i> step will allocate the first workspace used by the
```

Comment:
Sorry for this late comment, but there could be an option to select the Nth allocated workspace instead of the first one. Not important at this stage.